Server/Services/Implementations/SmtpService.cs

```diff
@@ -24,7 +24,7 @@ public async Task SendRegisterConfirmationEmailAsync(string email, string confir
         public async Task SendPasswordRecoveryEmailAsync(string email, string recoveryJwt, int idUser)
         {
             var smtpClient = CreateSmtpClient();
-            await smtpClient.SendMailAsync(_smtpConfigSection["Email"], email, "QUANDA - Recover password", $"https://localhost:44358/api/accounts/recover-password/{idUser}/{recoveryJwt}");
+            await smtpClient.SendMailAsync(_smtpConfigSection["Email"], email, "QUANDA - Recover password", $"https://localhost:44358/recover/password/reset?uuid={idUser}&recovery_token={recoveryJwt}");
```

Comment:
Suggestion - put link to new global variable so that it could be changed lately and to make code more readable